packages/ui-components/package.json

```diff
@@ -78,6 +78,6 @@
         "style-loader": "3.3.1",
         "ts-loader": "9.4.1",
         "typescript": "4.9.4",
-        "webpack": "5.74.0"
+        "webpack": "5.76.0"
```

Comment:
This change does not belong into this PR. Please put this in a separate one and describe that this is fixing a security issue

Reply:
Had not chance to do so, sorry.